doc/whats_new/v0.22.rst

```diff
@@ -7,7 +7,7 @@
 Version 0.22.0
 ==============
 
-**In Development**
+**November 29 2019**
```

Comment:
Are we sure about this?

Reply:
I hope so. I'm planning to do RC2 next week, and the release the week after that. Worst case we change it. We have in the past.